src/platform/landing-pages/header-footer-data.json

```diff
@@ -0,0 +1,321 @@
+      "title": "Accessibility",
+      "rel": "noopener noreferrer"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/orm/NOFEAR_Select.asp",
+      "order": 2,
+      "target": null,
+      "title": "No FEAR Act Data"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/oig/",
+      "order": 3,
+      "target": null,
+      "title": "Office of Inspector General"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/opa/Plain_Language.asp",
+      "order": 4,
+      "target": null,
+      "title": "Plain language"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/privacy-policy/",
+      "order": 5,
+      "target": null,
+      "title": "Privacy, policies, and legal information"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/privacy/",
+      "order": 6,
+      "target": null,
+      "title": "VA Privacy Service"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/foia/",
+      "order": 7,
+      "target": null,
+      "title": "Freedom of Information Act (FOIA)"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://www.usa.gov/",
+      "order": 8,
+      "target": "_blank",
+      "title": "USA.gov",
+      "rel": "noopener noreferrer"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/scorecard/",
+      "order": 9,
+      "target": null,
+      "title": "VA.gov scorecard"
+    },
+    {
+      "column": "bottom_rail",
+      "href": "https://staging.va.gov/veterans-portrait-project/",
+      "order": 10,
+      "target": null,
+      "title": "Veterans Portrait Project"
+    }
+  ],
+  "megaMenuData": [
+    {
+      "title": "VA Benefits and Health Care",
+      "menuSections": [
+        {
+          "title": "Health care",
+          "links": {
+            "seeAllLink": {
+              "text": "View all in health care",
+              "href": "https://staging.va.gov/health-care"
+            },
+            "columnOne": {
+              "title": "Get health care benefits",
+              "links": [
+                {
+                  "text": "About VA health benefits",
+                  "href": "https://staging.va.gov/health-care/about-va-health-benefits"
+                },
+                {
+                  "text": "How to apply",
+                  "href": "https://staging.va.gov/health-care/how-to-apply"
+                },
+                {
+                  "text": "Family and caregiver health benefits",
+                  "href": "https://staging.va.gov/health-care/family-caregiver-benefits"
+                },
+                {
+                  "text": "Apply now for health care",
+                  "href": "https://staging.va.gov/health-care/apply/application"
+                }
+              ]
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
```

Comment:
This is all just from Drupal

Reply:
Yeah, the idea here is that it's just a placeholder on the application landing pages for local development. There's something in `startApp` (or something) that required this data, so rather than refactor it, I figured I'd just feed it this placeholder for now. The next step is to move all that out, I think, but I don't know what that means yet.